File: app/routers/license.py
"""License stub endpoint for the Mock Jira Server (Server flavor).

Mounted under the /rest/plugins prefix in main.py, giving the full path:
    GET /rest/plugins/applications/1.0/installed/jira-software/license
"""

from fastapi import APIRouter
from fastapi.responses import JSONResponse

from app import shapes

router = APIRouter()


@router.get("/applications/1.0/installed/jira-software/license")
def license_():
    return JSONResponse(content=shapes.license_stub())

File: src/routes/license.ts
/**
 * License stub endpoint for the Mock Jira Server (Server flavor).
 *
 * Mounted under the /rest/plugins prefix in app.ts, giving the full path:
 *     GET /rest/plugins/applications/1.0/installed/jira-software/license
 */

import { FastifyInstance } from "fastify";

import * as shapes from "../shapes";

export async function licenseRoutes(fastify: FastifyInstance): Promise<void> {
  fastify.get("/applications/1.0/installed/jira-software/license", async (_request, reply) => {
    return reply.send(shapes.licenseStub());
  });
}
